test-framework/junit5-mockito/src/main/java/io/quarkus/test/junit/mockito/MockitoConfig.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.test.junit.mockito;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import io.quarkus.test.InjectMock;
+
+/**
+ * This annotation can be used to configure a Mockito mock injected in a field of a test class, i.e. a field annotated with
```

Comment:
Maybe mention that this only makes sense for `@QuarkusTest` and is useless/ignored for `@QuarkusComponentTest`?

Reply:
Probably makes sense...

Reply:
So this is totally about my understanding, not about the code, and not about this particular section of code, but.
 
 My impression - based totally on @mkouba's demo, not on actually reading the code - was that if you used `@QuarkusComponentTest`, you were setting the system under test to be narrow, and therefore other parts of the system were replaced with mocks, and that we'd default to using Mockito for those mocks. 

I have to admit I was always a bit hazy about where the mockito came from, if people didn't have the `junit-5-mockito` dependency on their classpath ... but if my broad understanding is correct, don't we need mechanisms to adjust the behaviour of mockito in the `@QuarkusComponentTest` scenario?

Reply:
> I have to admit I was always a bit hazy about where the mockito came from, if people didn't have the junit-5-mockito dependency on their classpath...

`quarkus-junit5-component` depends on mockito as well... no `quarkus-junit5-mockito` needed.

Reply:
Ah, got it. So if they wanted to use `@QuarkusComponentTest` and another mocking framework, they'd be 
- out of luck for the actual `@QuarkusComponentTest`s, which is reasonable
- maybe out of luck elsewhere if they try and use the new `@InjectMock, but since we haven't really written the pluggable back-end for that, maybe not?

Reply:
Done.